### PR TITLE
Aws e2e testing

### PR DESCRIPTION
**Pull request by EItanya**
BOT NOTES: 
resolves https://github.com/solo-io/supergloo/issues/327

### CODE REVIEW

File: pkg/install/utils/helm/helm_chart_installer_test.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ var (
 	kubeClient kubernetes.Interface
 )
 
-var _ = Describe("HelmChartInstaller", func() {
+var _ = XDescribe("HelmChartInstaller", func() {
```
Comment (rickducott) [inline pkg/install/utils/helm/helm_chart_installer_test.go]: you're probably tracking but remove before merge (same with the other)

Comment (EItanya) [reply pkg/install/utils/helm/helm_chart_installer_test.go]: Actually, this is purposeful as scott's PR removes the need for these tests and they take quite a bit of time

File: test/e2e/appmesh/e2e_suite_test.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+	rootCtx, cancel = context.WithCancel(context.TODO())
+	// create sg ns
+	_, err = kube.CoreV1().Namespaces().Create(&kubev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{Name: superglooNamespace},
+	})
+	Expect(err).NotTo(HaveOccurred())
+
+	// start supergloo
+	go func() {
```
Comment (marcogschmidt) [inline test/e2e/appmesh/e2e_suite_test.go]: Why do we need this?

File: test/e2e/appmesh/e2e_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+		ObjectMeta: metav1.ObjectMeta{
+			Name: basicNamespace,
+		},
+	})
+	Expect(err).NotTo(HaveOccurred())
+
+	_, err = kube.CoreV1().Namespaces().Create(&kubev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   namespaceWithInject,
+			Labels: map[string]string{"istio-injection": "enabled"},
```
Comment (marcogschmidt) [inline test/e2e/appmesh/e2e_suite_test.go]: Nit
```suggestion
			Labels: map[string]string{"app-mesh-injection": "enabled"},
```

**Pull request merged**